modules/hostap/src/supp_api.c

```diff
@@ -2408,6 +2408,7 @@ int supplicant_ap_wps_pbc(const struct device *dev)
 
 int supplicant_ap_wps_pin(const struct device *dev, struct wifi_wps_config_params *params)
 {
+#define WPS_PIN_EXPIRE_TIME 120
```

Comment:
I do not understand the commit message. Is this parameter a timeout while waiting the user to type the pin?

Reply:
Can we use full for `author_macs` -> `Authorized MAC IE`? I was searching the code for `author_macs` :).
A reword would help understand.

```
hostap: wps: Enable PIN expiry timeout

Enable WPS PIN expire timeout parameter, this helps use in cleanup of the Authorized
MAC IE in the beacon in case no peer is connected within the timeout. Without this
parameter the  IE is not removed from the beacon.
```
WDYT?

Reply:
okay, I have updated the comment as you offer.

Reply:
`use` -> `us`

Reply:
updated, use -> us